Sources/Build/BuildPlan.swift

```diff
@@ -576,13 +583,17 @@ public final class SwiftTargetBuildDescription {
         guard let bundlePath = self.bundlePath else { return }
 
         let stream = BufferedOutputByteStream()
+
+        let mainPath: AbsolutePath =
+            AbsolutePath(Bundle.main.bundlePath).appending(component: bundlePath.basename)
+
         stream <<< """
         import class Foundation.Bundle
 
         extension Foundation.Bundle {
             static var module: Bundle = {
-                let mainPath = Bundle.main.bundlePath + "/" + "\(bundlePath.basename)"
```

Comment:
This change actually broke the lookup for bundle resources. Since `Bundle.main` is now executed inside the `swift build` context, `mainPath` is now `/usr/bin` on linux instead of the location of the output binary.

Reply:
I've moved the call to `Bundle.main` back to the generated code in #3463